backend/benefit/applications/management/commands/check_drafts_to_delete.py

```diff
@@ -0,0 +1,78 @@
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+from django.utils.text import format_lazy
+from django.utils.translation import gettext_lazy as _
+
+from applications.enums import ApplicationStatus
+from applications.models import Application
+from messages.automatic_messages import send_email_to_applicant
+
+APPLICATION_ABOUT_TO_BE_DELETED_MESSAGE = _(
+    "Your application {id} will be deleted soon. If you want to continue the application process, please do so by "
+    "{application_deletion_date}, otherwise the application will deleted permanently."
+)
+
+
+class Command(BaseCommand):
+    help = "Query draft applications that are close to the deletion date and send a notification to the applicant"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--days_to_deletion",
```

Comment:
Using `-` as an intermediate character would make the command line argument naming be more in line with Django utilities and CLI programs in general, e.g., `--days-to-deletion`.

Reply:
Yeah I would like to name the arguments like this, but Pylance does not like it, when the arguments are used as a parameters for the jobs.
<img width="591" alt="Screenshot 2023-02-16 at 9 03 53" src="https://user-images.githubusercontent.com/33894149/219292283-dfa91276-51ea-409b-80d2-459be2dd3a94.png">

Reply:
Damn, of course :smile: